charts/k8s-monitoring/destinations/otlp-values.yaml

```
@@ -229,6 +229,24 @@ processors:
       resource: []
       span: []
       spanevent: []
+
+  filter:
```

```suggestion
  filters:
```
we should match what we have in feature-application-observability/values.yaml

---

This one I agree with.